source/cloud-security/amazon/services/troubleshooting.rst

```diff
@@ -13,7 +13,7 @@ The below information is intended to assist in troubleshooting issues.
 Checking if the module is running
 ---------------------------------
 
-When the module runs it writes its output in the ``ossec.log`` file. This log file can be found in ``WAZUH_PATH/logs/ossec.log`` or under **App Logs** if using the Wazuh UI.
+When the module runs it writes its output in the ``ossec.log`` file. This log file can be found in ``WAZUH_PATH/logs/ossec.log`` or under **Logs** if using the Wazuh UI.
```

Comment:
```suggestion
When the module runs it writes its output in the ``ossec.log`` file. This log file can be found in ``WAZUH_PATH/logs/ossec.log`` and under **Logs** if using the Wazuh UI.
```

Reply:
I added the suggestion and clarified where the `Logs` application can be found.